src/pages/Main/ImageSlider/FirstSlider.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import styled from 'styled-components';
+import FIRST_DATAS from './FIRST_DATAS';
+
+function FirstSlider() {
+  const settings = {
+    dots: true,
+    infinite: true,
+    speed: 200,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+
+  return (
+    <Container>
+      <StyleSlide {...settings}>
+        {FIRST_DATAS.map(DATA => (
+          <div key={DATA.id}>
+            <Item>
+              <Image src={DATA.img} alt="제품이미지" />
+            </Item>
+          </div>
+        ))}
+      </StyleSlide>
+    </Container>
+  );
+}
+
+export default FirstSlider;
+
+const Container = styled.div`
+  margin: 0 auto;
+  width: 500px;
+`;
+const StyleSlide = styled(Slider)`
+  .slick-list {
+    justify-content: center;
+    height: 300px;
+    background-color: darkgoldenrod;
+  }
+`;
```

Comment:
```suggestion
const Container = styled.div`
  margin: 0 auto;
  width: 500px;
`;

const StyleSlide = styled(Slider)`
  .slick-list {
    justify-content: center;
    height: 300px;
    background-color: darkgoldenrod;
  }
`;
```
css에서 다른 selector들 끼리는 공백을 부여해주세요!
사소한 부분이라고 간과할 수 있지만 이런 부분들이 모여서 코드의 가독성을 좋게 만듭니다